redshift/resource_redshift_schema.go

```diff
@@ -120,9 +134,13 @@ func readRedshiftSchema(d *schema.ResourceData, db *sql.DB) error {
 	var (
 		schemaName string
 		owner      int
+		quota      int
 	)
 
-	err := db.QueryRow("select nspname, nspowner from pg_namespace where oid = $1", d.Id()).Scan(&schemaName, &owner)
+	err := db.QueryRow(`
+			select trim(nspname) as nspname, nspowner, coalesce(quota, 0) as quota from pg_namespace
```

Comment:
Super nitpick: could we move `from pg_namespace` to the next line so it's just a little more clear how this query is structured?

Reply:
c4ea12b02068